boards/microchip/pic32cxsg41_cult/pic32cxsg41_cult.dts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2024 Microchip
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <microchip/pic32cx1025sg.dtsi>
+#include "pic32cxsg41_cult-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "PIC32CX SG41 Curiosity Ultra";
+	compatible = "pic32cxsg41,cult", "microchip,pic32cx1025sg41128", "microchip,pic32cxsg";
+
+	chosen {
+		zephyr,console = &sercom4;
+		zephyr,shell-uart = &sercom4;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		userled1 = &led1;
+		userled2 = &led2; 
+		pwm-led1 = &pwm_led1;
+		sw1 = &button1;
+		sw2 = &button2;
+		i2c-0 = &sercom7;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led1: led_1 {
```

Comment:
newline needed here and for spacing between child nodes

Reply:
```
		compatible = "gpio-leds";
+
		led1: led_1 {
```